docs/customer-portal/cp-dashboard.md

```diff
@@ -1,44 +1,46 @@
 # Dashboard
+
 The **Customer Portal** provides access to configurations, real-time testing and stats, and more information about your account. The **Dashboard** provides an overview of Call Stats and Summaries.
 
 ## Top of Page Functions
+
 ![alt text][cp-dash]
 
-+ **Checklist** A Quick view of outstanding Alerts on the account. Upon initial setup, Alerts are sent to verify email and mobile for the account, and a reminder to create customers and setup carriers. 
++ **Checklist** A Quick view of the outstanding Alerts on the account. Upon initial setup, verification of email and mobile for the account made through Alerts, and a reminder is put across to create customers and setup carriers.
 
-+ **Time Zone** Click on the Date/Time box to select the Time Zone for your site. 
++ **Time Zone** Click on the Date/Time box to select the Time Zone for your site.
 
-+ **Channel Count** Total Live Channels currently in use and total configured Channels.
++ **Channel Count** Total Live Channels presently in use and total configured Channels.
 
-+ **Account Balance** Displays the current account balance and a link to make a payment using a credit card or PayPal. 
++ **Account Balance** Displays the current account balance and a link to make a payment using a credit card or PayPal.
 
 + **Account** Use this to view your Company Profile, view and update your personal Profile, Change Password or Language, or Log Out.
 
-
 !!! warning "Account Balance functions"
-    The "Click to Make Payment" link is currently in development. Please navigate to **Payment** on the left-hand menu for this functionality. 
-
+    The "Click to Make Payment" link is presently under development. Please navigate to **Payment** on the left-hand menu for this functionality.

```

Comment:
This feature is live and ready for use, please remove this Warning

Reply:
removed the warning

please check

https://bani-customerportal-2022-12-29--connexcs-docs.netlify.app/customer-portal/cp-dashboard/